shared/lib_irradproc.cpp

```diff
@@ -363,6 +363,10 @@ void incidence(int mode,double tilt,double sazm,double rlim,double zen,double az
 				rot = backrot * M_PI/180; // convert backtracked rotation angle to radians
 			}
 
+			//optionally force the tracker to a "stow" angle if specified
+			if (force_to_stow)
+				rot = stow_angle_deg * DTOR;
+
```

Comment:
if force_to_stow is true then it'll overwrite the backtracking angle, so I would check for force_to_stow first and if it's true skip the backtracking calculation:

if (force_to_stow) {...}
else if (en_backtrack) {...}

Reply:
Good point! Updated.